filebeat/module/kafka/log/manifest.yml

```diff
@@ -0,0 +1,15 @@
+module_version: 1.0
+
+var:
+  - name: kafka_home
+    default:
+      - /opt/kafka[_-\.0-9]*a
+  - name: paths
+    default:
+      - ${kafka_home}/logs/controller.log*
```

Comment:
Does this work? I would have expected this to use go template syntax, so something like `{{.kafka_home}}`.